src/sources/docker_logs.rs

```diff
@@ -615,7 +613,7 @@ impl EventStreamBuilder {
             .filter_map(|v| ready(v.unwrap()))
             .take_until(self.shutdown.clone());
 
-        let events_stream: Box<dyn Stream<Item = Event> + Unpin + Send> =
+        let events_stream: Box<dyn Stream<Item = LogEvent> + Unpin + Send> =
```

Comment:
I like using the more direct type here.

Reply:
Yeah, at some point it may be possible to drop `enum Event` entirely from large parts of the code base (though obviously not everywhere, VRL comes to mind).